app/views/early_bird_dashboard/index.html.slim

```diff
@@ -63,4 +64,13 @@
               - else
                 span.constituency #{question.member_constituency}
             - if feature_propose_ao_enabled?
-              p = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
+              - if question.proposed?
+                h3 Deputy Director(s) proposed to answer:
+                p.action_officers-list
+                  - names = question.action_officers.uniq.pluck :name
+                  = names.join(', ')
+
+              p.propose-links
+                = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
+                = ' | '
+                = link_to('Propose a Deputy Director', new_pq_proposal_path(question.id))
```

Comment:
If this is related to the question then maybe change to Propose a Deputy Director for Question XXX"

Reply:
Accessibility issue - I will create tickets for these.  Not a blocker